user-org-service/app/controllers/BaseController.java

```diff
@@ -23,13 +25,17 @@
   @Inject private HttpExecutionContext httpObject;
 
   public String getDummyResponse() {
-    String dummyResponse =
+      String dummyResponse =
         "{\"id\":\"api.user.200ok\",\"ver\":\"v1\",\"ts\":\"2019-01-17 16:53:26:286+0530\",\"params\":{\"resmsgid\":null,\"msgid\":\"8e27cbf5-e299-43b0-bca7-8347f7ejk5abcf\",\"err\":null,\"status\":\"success\",\"errmsg\":null},\"responseCode\":\"OK\",\"result\":{\"response\":{\"response\":\"SUCCESS\",\"errors\":[]}}}";
+    ProjectLogger.log(
+            "BaseController : getDummyResponse : "+ dummyResponse, LoggerEnum.INFO.name());
```

Comment:
Use same pattern for all logger (className:methodName:message) don't put colon with different tab.